components/webhook-mgt/org.wso2.carbon.identity.webhook.management/src/main/java/org/wso2/carbon/identity/webhook/management/api/core/cache/ActiveWebhooksCache.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.webhook.management.api.core.cache;
```

Comment:
Seems like we haven't exported core package from the component in the pom.

https://github.com/wso2/carbon-identity-framework/blob/18f0046eec793ff1c5968bbb3edead1db1023d97/components/webhook-mgt/org.wso2.carbon.identity.webhook.management/pom.xml#L130C25-L134C42

Refer: https://github.com/wso2/carbon-identity-framework/blob/5aab1c16652e5ce73a0c6fbb2573638d40925eda/components/action-mgt/org.wso2.carbon.identity.action.management/pom.xml#L101